trestle/core/validator_helper.py

```diff
@@ -229,27 +117,27 @@ def regenerate_uuids_in_place(object_of_interest: Any, uuid_lut: Dict[str, str])
                 new_dict[uuid_str] = new_val
                 uuid_lut[value] = new_val
             else:
-                new_value, uuid_lut = regenerate_uuids_in_place(value, uuid_lut)
+                new_value, uuid_lut = _regenerate_uuids_in_place(value, uuid_lut)
                 new_dict[key] = new_value
         return new_dict, uuid_lut
     return object_of_interest, uuid_lut
 
 
-def update_new_uuid_refs(object_of_interest: Any, uuid_lut: Dict[str, str]) -> Tuple[Any, int]:
+def _update_new_uuid_refs(object_of_interest: Any, uuid_lut: Dict[str, str]) -> Tuple[Any, int]:
```

Comment:
This function contains a [code smell](https://sonarcloud.io/project/issues?id=compliance-trestle&open=AX5rlNIt7Nfw18hwjlm3&pullRequest=993&resolved=false&sinceLeakPeriod=true&types=CODE_SMELL), do you think we can somehow simplify it?